internal/convert/registryv1.go

```diff
@@ -321,6 +321,9 @@ func Convert(in RegistryV1, installNamespace string, targetNamespaces []string)
 	}
 	for _, obj := range in.Others {
 		obj := obj
```

Comment:
Should we also do the `IsSupported` check?

https://github.com/operator-framework/operator-registry/blob/87c0b28d82eb882c344ba2c7efc696063c227a67/pkg/lib/bundle/supported_resources.go#L53

Reply:
Do we want to? One of the benefit of v1 was to not restrict users with the specific kind of GVKs being supported